tests/utils/test_task_group.py

```diff
@@ -559,3 +559,19 @@ def test_duplicate_group_id():
             _ = DummyOperator(task_id="task1")
             with TaskGroup("group1"):
                 _ = DummyOperator(task_id="upstream_join_id")
+
```

Comment:
One more empty line is needed here to please static check.

On the other hand, I agree with you on https://github.com/apache/airflow/issues/12757#issuecomment-739147569 . Will do the discussion there.

Reply:
Thanks!